test/vector-tile.test.js

```diff
@@ -805,6 +816,14 @@ describe('mapnik.VectorTile ', function() {
             done();
         });
     });
+
+    it('should return the correct bufferedExtent', function(done) {
+        var vtile = new mapnik.VectorTile(9,112,195);
+        var extent = vtile.bufferedExtent();
+        var expectedExtent = [-11273544.427724076, 4693845.032936104, -11190380.940949803, 4777008.519710373];
```

Comment:
@flippmoke what's the best way to avoid precision checks like this? Sometimes rounding errors lead to failures between `clang` and `clang XCODE`. https://travis-ci.org/mapnik/node-mapnik/jobs/108136046#L1922